ui/imodel-components-react/src/imodel-components-react/timeline/SolarTimeline.tsx

```diff
@@ -544,18 +543,18 @@ export class SolarTimeline extends React.PureComponent<
             <Label htmlFor="speed">Timeline speed</Label>
           </VisuallyHidden>
           <Tooltip content={this._speedLabel}>
-            <select
-              className="solar-timeline_speed"
+            <Select
+              native
+              styleType="borderless"
               name="speed"
-              onChange={(e) => this._onSpeedChange(Number(e.target.value))}
+              onChange={(newValue) => this._onSpeedChange(Number(newValue))}
               value={currentSpeed}
-            >
-              {this._speeds.map((speed) => (
-                <option key={speed} value={speed}>
-                  {speed}x
-                </option>
-              ))}
-            </select>
+              options={this._speeds.map((speed) => ({
+                key: speed,
```

Comment:
is there a reason for the key prop here? My understanding is that it's used primarily for JSX elements

Reply:
This is unnecessary and can be removed.

Reply:
Removed `key` property.